spec/lib/telephony/pinpoint/sms_sender_spec.rb

```diff
@@ -31,6 +31,10 @@ def ==(other)
       Pinpoint::MockClient.message_response_result_status_message = status_message
     end
 
+    after do
+      Telephony::Pinpoint::SmsSender::CLIENT_POOL.clear
+    end
```

Comment:
what if we did this with an `around` block, so we can still add an `ensure` and don't need to add to each spec below?

```suggestion
    around do |ex|
      ex.run
    ensure
      Telephony::Pinpoint::SmsSender::CLIENT_POOL.clear
    end
```

Reply:
good call, changed in the file in 66f494f93